src/components/LimitMessage.js

```diff
@@ -0,0 +1,29 @@
+import { Link } from "react-router-dom";
+import errorImg from "../assets/error-image.png";
+import BookIcon from "../assets/book-icon.png";
+
+const LimitMessage = () =>{
+    return(
+        <div className="error-container">
+        <img src={errorImg} className="error-container-img" alt="error message"/>
+            <div className="error-message">
+                <span>Oops!</span>
+                <p className="limit-message">
+                You have reached the maximum number of books in your collection!
+                </p>
+                <p>
+                Remove books before you can add more.
+                </p>
+
+                <Link to="/bookshelf">
+                    <div className="bookshelf-link">
+                        <img src={BookIcon} alt="books icon" />
```

Comment:
Not sure if it would also be slightly more accessible here to describe what the books icon does (looks like it links back to /bookshelf eh?).